### DISCUSSION TITLE
This calibration script doesn't show on Cura ater restart

### DISCUSSION THREAD

**Discussion by EA7X** [5axes/Calibration-Shapes#126] [General]
On cura.log:
ERROR - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [217]: Script CalibrationShapes.py is not a recognised script type. Ensure it inherits Script

**Comment by 5axes** [comment]
CalibrationShapes.py  is not a PostProcessing Scripts but the source code of the plugin.  I don't know what you are trying to do.

**Comment by EA7X** [comment]
Sorry, I'm not a skillful 3D print user. Just was trying to find something
to calibrate my Ender 3 pro. I thought this would help.

Season greetings

José
